Sources/Controllers/Panels/OAMapPanelViewController.mm

```diff
@@ -779,15 +779,37 @@ - (void) closeRouteInfo:(void (^)(void))onComplete
     }
 }
 
+- (void) closeRouteInfoForSelectPoint:(void (^)(void))onComplete
+{
+    if (self.routeInfoView.superview)
+    {
+        [self setBottomControlsVisible:YES menuHeight:0 animated:YES];
+        
+        [self.routeInfoView hide:YES duration:.2 onComplete:^{
+            [self setTopControlsVisible:YES];
+            [_hudViewController.quickActionController updateViewVisibility];
+            if (onComplete)
+                onComplete();
+        }];
+        
+        [self destroyShadowButton];
+        
+        self.sidePanelController.recognizesPanGesture = NO; //YES;
+    }
+}
+
 - (CGRect) shadowButtonRect
 {
     return self.view.frame;
 }
 
 - (void) removeGestureRecognizers
 {
-    while (self.view.gestureRecognizers.count > 0)
-        [self.view removeGestureRecognizer:self.view.gestureRecognizers[0]];
+    for (UIGestureRecognizer *recognizer in self.view.gestureRecognizers)
```

Comment:
Check if the panel cannot be dragged while other screens (like context menu) are presented

Reply:
https://www.dropbox.com/s/amyxyr7hway7x39/Screen%20Recording%202021-08-05%20at%2016.25.19.mov?dl=0